.github/workflows/continuous-deployment.yml

```diff
@@ -25,6 +23,8 @@ jobs:
         run: dotnet publish Epsilon.Cli --runtime ${{ matrix.RID }} --configuration Release -p:PublishSingleFile=true -p:PublishTrimmed=true --self-contained true
       - name: Copy appsettings.json
         run: cp /home/runner/work/epsilon/epsilon/Epsilon.Cli/appsettings.example.json /home/runner/work/epsilon/epsilon/Epsilon.Cli/bin/Release/net6.0/${{ matrix.RID }}/publish/appsettings.json
```

Comment:
Can you use work directory instead of the full path, makes reading a little easier

Reply:
I don't know whether GitHub actions has native support for retrieving the working directory, however you can also use this: https://aws1.discourse-cdn.com/github/original/1X/07b48855ef4a2d6b635f7342302384baeaa3b728.png